test/fixtures/keys/agent10.cnf

```diff
@@ -0,0 +1,17 @@
+[ req ]
+default_bits           = 1024
+days                   = 9999
+distinguished_name     = req_distinguished_name
+attributes             = req_attributes
+prompt                 = no
+
+[ req_distinguished_name ]
+C                      = US
+ST                     = CA
+L                      = SF
+O                      = Joyent
```

Comment:
Can you use "The Node.js Foundation" or something along those lines?

Reply:
ok